pkg/pipelines/eventlisteners/eventlisteners.go

```diff
@@ -122,3 +121,12 @@ func createListenerObjectMeta(name, ns string) metav1.ObjectMeta {
 		Namespace: ns,
 	}
 }
+
+func createBindings(bindings []string) []*triggersv1.EventListenerBinding {
+	createBindingSlice := []*triggersv1.EventListenerBinding{}
```

Comment:
How about we make this a bit nicer?

The thing that's passed in is not really a set of bindings, but the _names_ for those bindings.

And, we know in advance how many items there will be.

```go
func createBindings(names []string) []*triggersv1.EventListenerBinding {
    bindings := make([]*triggersv1.EventListenerBinding, len(names))
    for i, name := range names {
        bindings[i] = createListenerBinding(name)
    }
    return bindings
}
```

Reply:
so instead of for loop get the numbers?

Reply:
Not quite, we have a slice of strings, and we're outputting a slice of EventListenerBindings, both are the same size.

So, we can optimise the code a bit (drop the append) because the first element in the output slice is directly related to the first element in the input slice.

In the original code, you're doing an enumeration over the input slice, but, you're dropping the number, the change simplifies the code a bit.

Reply:
sure! this sounds good